### DISCUSSION TITLE
About  bus_driver_scheduling_flow_sat.py

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#3984] [CP-SAT questions] [answered]
Could I know if below sample and data sets come from a paper or not?
 https://github.com/google/or-tools/blob/main/examples/python/bus_driver_scheduling_flow_sat.py
If yes, could you please share with me the title of the paper, thanks a lot!

John

**Comment by lperron** [comment, marked as answer]
No, they were donated by a CP-SAT user.
